content/en/docs/components/pipelines/quickstart.md

```diff
@@ -3,4 +3,332 @@ title = "Quickstart"
 description = "Getting started with Kubeflow Pipelines"
 weight = 3
                     
-+++
\ No newline at end of file
++++
+<!-- TODO: add links to more thorough content docs -->
+<!-- TODO: add UI screenshots for final pipeline -->
+In this tutorial we will deploy a Kubernetes cluster, deploy a KFP standalone deployment into that cluster, create and run a simple pipeline using the KFP SDK, view the pipeline on the KFP Dashboard, then create a more involved machine learning pipeline that uses additional KFP features.
+
+### 1) Deploy a Kubernetes cluster
```

Comment:
Should we skip the deployment part or reference the existing deployment instruction instead to avoid duplicate and potentially inconsistent content? 

/cc @zijianjoy

Reply:
Thanks for bringing this up. I can see it either way.

My original thinking was that a "Quickstart" section might be the only place where it is okay to have duplicative content, in the spirit of making the quickstart completely standalone and to prevent users from going down other documentation rabbit holes. I'm curious what a TW's perspective on this would be.

Reply:
Thanks for explaining your rationale. I'm convinced by this and will update.